docs/go-storage/internal/how-types.NewObject-works.md

```diff
@@ -0,0 +1,42 @@
+---
+title: How types.NewObject(client, done) works
+---
+
+### Introduction
+
+We use `types.NewObject(client, done)` to create a new object with client in services. 
+
+> func NewObject(client Storager, done bool) *Object
+
+**Parameters**
+
+client - the client in which `Object` is alive.
+
+done - marks whether the object stated. `true` specifies this object already stated, we don't need to `stat` it anymore.

```

Comment:
What about `stat`ed ?

Reply:
`stat` for the object.
I've updated the description, please help to check.